public/javascripts/SVValidate/src/mission/Mission.js

```diff
@@ -83,12 +85,19 @@ function Mission(params) {
      *                      progress will increase.
      */
     function updateMissionProgress(skip) {
+        // console.log(getProperties());
```

Comment:
Can you remove this comment?

Reply:
Do you want me to uncomment it, or just delete the line?

Reply:
Just delete the line